File: 백준/Silver/1978. 소수 찾기/소수 찾기.java
import java.util.ArrayList;
import java.util.Arrays;
import java.util.Scanner;

public class Main {
	public static void main(String[] args) {
			
		Scanner sc = new Scanner(System.in);
		
		int T = sc.nextInt();
        int[] num = new int[T];

        // Input Data
        for(int i=0; i<T; i++) {
            num[i] = sc.nextInt();
        }
        
        int n = num.length;
        ////////////////////////
		ArrayList<Integer> ans = new ArrayList<Integer>(); 
        	int total = 0;
        	for(int i=0; i<n; i++) {
        		int cnt = 0;
        		
        		for(int j=1; j<=num[i]; j++) {
        			if(num[i] % j == 0) {
        				cnt ++;
        				if(cnt>2) {
        					break;
        				}
        			}
        		}
        		
    			if(cnt==2) {
    				total++;
    			}
        	} // 
     		System.out.println(total);
        
		
	} //
} // 

File: 백준/Bronze/1978. 소수 찾기/소수 찾기.py
from math import ceil,sqrt

t = int(input())
num = list(map(int, input().split()))
ans = []

for i in num:
  cnt = 0
 
  if i == 1:
    continue
  if i == 2:
    ans.append(i)

  for j in range(2,ceil(sqrt(i))+1):
    if i % j  == 0:
      cnt += 1
  
  if cnt == 0:
    ans.append(i)
  

print(len(ans))
